lib/ansible/plugins/inventory/__init__.py

```diff
@@ -226,6 +227,31 @@ def _consume_options(self, data):
             if k in data:
                 self._options[k] = data.pop(k)
 
+    def _expand_hostpattern(self, hostpattern):
+        '''
+        Takes a single host pattern and returns a list of hostnames and an
+        optional port number that applies to all of them.
+        '''
+        # Can the given hostpattern be parsed as a host with an optional port
+        # specification?
+
+        try:
+            (pattern, port) = parse_address(hostpattern, allow_ranges=True)
+        except Exception:
+            # not a recognizable host pattern
+            pattern = hostpattern
```

Comment:
cleaner:
```python
pattern, port = hostpattern, None
```

Reply:
> Multiple assignment is slower than individual assignment. For example "x,y=a,b" is slower than "x=a; y=b".

https://wiki.python.org/moin/PythonSpeed#Take_advantage_of_interpreter_optimizations

Also, when not necessary, I'm not going to update code that I don't need to, especially when just moving it from one location to another.

My goal is to touch as small amount of code when making changes or adding features.